### PR TITLE
Feature footer/contact devs

### PR DESCRIPTION
**Pull request by lbmerchant93**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [x] Adds a new feature
- [x] Improves an existing feature
- [x] Fixes a bug
- [x] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
Includes redirects for user flow. Include Contact the Devs page. Fix NavBar links for modules and courses. 

## Checklist:
- [x] I have performed a self-review of my own code
- [ ] I have run Lighthouse/Wave checks for accessibility
- [x] New and existing unit tests pass locally with my change

### CODE REVIEW

File: context/app-context.js [hunk 1]
```diff
@@ -21,7 +21,6 @@ export function AppWrapper({ children }) {
         "#ff2500"
       ]
     const [hasBeenUpdated, setHasBeenUpdated] = useState(false)
-    const [hasBeenDeleted, setHasBeenDeleted] = useState(false)
```
Comment (alia-peterson) [inline context/app-context.js:24]: glad we were able to consolidate this!

File: cypress/integration/dashboard_spec.js [hunk 2]
```diff
@@ -5,26 +5,25 @@ context('Dashboard View', () => {
 
   beforeEach(() => {
     cy.intercept('GET', coursesApi, { fixture: 'courses-api' })
-    cy.visit(baseUrl)
+    cy.visit(baseUrl).wait(300)
```
Comment (alia-peterson) [inline cypress/integration/dashboard_spec.js:8]: Just curios why this was needed, i was under the impression cypress was built to not move forward until an item is found with the `find` command so i didn't think we needed this. would love to know more info!

File: pages/addCourseForm.js [hunk 3]
```diff
@@ -51,11 +63,10 @@ export default function addCourseForm() {
 
     } else {
       setHasBeenUpdated(!hasBeenUpdated)
-      const newCourseIndex = sharedState.courses.length - 1
-      const newCourseId = sharedState.courses[newCourseIndex].id
-      setSharedState({...sharedState, currentCourse: newCourseId})
+      router.push('/courseDashboard')
```
Comment (alia-peterson) [inline pages/addCourseForm.js:66]: This is a great solution for routing to the correct page after the course is created!

**Review by alia-peterson — APPROVED** [review]

File: pages/contactDevelopers.js [hunk 4]
```diff
@@ -0,0 +1,101 @@
+import styles from '../styles/contactDevelopers.module.scss'
+import Image from 'next/image'
+
+const ContactDevelopers = () => {
+
```
Comment (srslie) [inline pages/contactDevelopers.js:5]: So cute! Thank you!

**Review by srslie — APPROVED** [review]
Thanks for adding that!

**Pull request merged**